test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
I'm using a custom function that sets `protocol_required`. See https://github.com/18F/federalist/pull/1049/files#diff-5a61a2437a3a240173dd03188316466aR48